src/common-utils/main.sh

```diff
@@ -551,7 +551,11 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
 
         # Add devcontainer .zshrc template
         if [ "$INSTALL_OH_MY_ZSH_CONFIG" = "true" ]; then
```

Comment:
Can you add tests to validate the changes?

Reply:
test added